cl/lib/utils.py

```diff
@@ -104,3 +102,23 @@ def human_sort(
         sorter = lambda item: [convert(c) for c in re.split("([0-9]+)", item)]
 
     return sorted(unordered_list, key=sorter)
+
+
+def wrap_text(length: int, text: str) -> str:
+    """Wrap text to specified length without cutting words
+
+    :param length: max length to wrap
+    :param text: text to wrap
+    :return: text wrapped
+    """
+    words = text.split(" ")
+    if words:
+        lines = [words[0]]
+        for word in words[1:]:
+            if len(lines[-1]) + len(word) < length:
+                lines[-1] += f" {word}"
+            else:
+                lines.append(word)
+                break
+        return " ".join(lines)
+    return ""
```

Comment:
was this just moved from somewhere else?

Reply:
I think it would be more useful to have it here, since although it is only used in lookup_utils.py, this function could be used in other places